schemas/description.json

```diff
@@ -1,10 +1,13 @@
 {
   "$id": "https://hbz.github.io/lobid/schemas/description.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "title": "Description",
+  "title": "Beschreibung",
   "description": "The description of the object e.g. product, project. MUST have german and english Version",
   "allOf": [
     { "$ref": "https://hbz.github.io/lobid/schemas/localizedString.json" },
     { "required": ["en", "de"]}
```

Comment:
If the language is limited to en and ger the required-condition kann be integrated to the localizedString

Reply:
I will rather remove the `required` for now so that we can focus on a German version first.